modules/vpc/main.tf

```diff
@@ -39,3 +40,27 @@ module "vpc" {
     "kubernetes.io/cluster/${var.kubernetes_cluster_name}" = "shared"
   }
 }
+
+module "nat_instance" {
+  # create nat instance instead of nat gateway
+  count  = var.enable_nat_gateway ? 0 : 1
+  source = "int128/nat-instance/aws"
+
+  name = "${var.project}-${var.environment}"
+
+  vpc_id                      = module.vpc.vpc_id
+  public_subnet               = module.vpc.public_subnets[0]
+  private_subnets_cidr_blocks = module.vpc.private_subnets_cidr_blocks
+  private_route_table_ids     = module.vpc.private_route_table_ids
+
```

Comment:
How about exposing the `instance_types` array of the module as a variable so people can override if necessary?